src/pkg/api/composition.go

```diff
@@ -0,0 +1,96 @@
+package api
+
+import (
+	"encoding/json"
+	"hms/gateway/pkg/config"
+	"hms/gateway/pkg/docs/service/composition"
+	"hms/gateway/pkg/docs/types"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"hms/gateway/pkg/docs/model"
+	"hms/gateway/pkg/docs/service"
+	"hms/gateway/pkg/errors"
+)
+
+type CompositionHandler struct {
+	service *composition.CompositionService
+}
+
+func NewCompositionHandler(docService *service.DefaultDocumentService, cfg *config.Config) *CompositionHandler {
+	return &CompositionHandler{
+		service: composition.NewCompositionService(docService, cfg),
+	}
+}
+
+func (h CompositionHandler) Create(c *gin.Context) {
```

Comment:
Need to make a description for swagger similar to other routes

Reply:
\\+